webpack.config.js

```diff
@@ -375,7 +375,7 @@ module.exports = {
     resolve: {
         alias: aliases,
         modules: [ libdir, nodedir ],
-        extensions: ["*", ".js", ".json", ".less"]
+        extensions: ["*", ".js", ".json"]
```

Comment:
Should `.scss` be here as well? `.less` was before.

Reply:
This was for imports. So that you can address files from pkg/lib or node_modules without relative imports  but just by addressing the name of the file.
Since we don't allow that for css files I don't see the reason to do that for scss.